README.md

```diff
@@ -161,8 +161,26 @@ external leftpad : string -> int -> char -> string = "" [@@bs.val] [@@bs.module
 
 ## Browser-specific
 
-#### Extract all links form a webpage
+#### Extract all links from a webpage
 
+```ml
+open ReasonJs.Dom
+open Document
+
+let all_links () =
+  document
+  |> querySelectorAll "a"
```

Comment:
Indeed, that `open Document` caused me some problem before when I used `ofNode` and got a `Document` of a node instead of an `Element`...